Missing launch file for moveit

Hi everyone,

I installed the Gazebo simulator as per instructions, using Ubuntu 20.04 and ROS2 Galactic. Launching `ros2 launch ur_simulation_gazebo ur_sim_control.launch.py` works like a charm. I then try to launch the test_controller as described with `ros2 launch ur_bringup test_joint_trajectory_controller.launch.py` and I don't get anything moving. as I get the output from this node: [https://pastebin.com/p028hS6Z](https://pastebin.com/raw/p028hS6Z)

But, more importantly (for me) is that when I try to launch the configuration with gazebo and MoveIt with `ros2 launch ur_simulation_gazebo ur_sim_moveit.launch.py` I get: [https://pastebin.com/LQqBprDy](https://pastebin.com/raw/LQqBprDy)

And, fair enough, I get no launch folder there: https://pastebin.com/raw/LWGXwzrp

> Does scaled controller does not work with Gazebo? Or what's exact issue?

Yes, exactly. Starting
```
ros2 launch ur_simulation_gazebo ur_sim_control.launch.py launch_rviz:=false initial_joint_controller:=scaled_joint_trajectory_controller
```
yields the error
```
[gzserver-4] [ERROR] [1645691788.270588178] [controller_manager]: Can't activate controller 'scaled_joint_trajectory_controller': State interface with key 'speed_scaling/speed_scaling_factor' does not exist
```

I tried removing the `unless simulation` [block](https://github.com/UniversalRobots/Universal_Robots_ROS2_Description/blob/d739cb6291ec3f8b4c645022a5eb32b2bc1f20cf/urdf/ur.ros2_control.xacro#L154) in the description. While gazebo starts fine without that `unless` statement, the error stays when using the scaled jtc.

--- a/ur_simulation_gazebo/launch/ur_sim_moveit.launch.py
+++ b/ur_simulation_gazebo/launch/ur_sim_moveit.launch.py
@@ -79,6 +79,7 @@ def launch_setup(context, *args, **kwargs):
             "prefix": prefix,
             "use_sim_time": "true",
             "launch_rviz": "true",
+            "use_mock_hardware": "true",
         }.items(),
     )
 

